client/src/Navbar.jsx

```diff
@@ -42,7 +42,7 @@ var NavbarContainer = styled.nav`
   justify-content: space-between;
   background-color: #3a606e;
   margin: 1px;
-  height: 50px;
+  height: 3.5vw;
```

Comment:
It is kind of weird to use `vw` for height (because `vw` is literally "viewport width"). Thus I want to double check and make sure that you are indeed intending to scale the height based on the viewport width.